src/philemon/convert.go

```diff
@@ -0,0 +1,79 @@
+package philemon
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+const (
+	// APIKEY is the required apikey to make a request to the third-party
+	// conversion provider.
+	APIKEY = "1KwHRv7TVnTE7HHIJIanuq4rT7ilBPWq"
+	// convertCurrencyEndpoint is the currency conversion endpoint.
+	convertCurrencyEndpoint = "https://api.apilayer.com/fixer/convert?to=%s&from=%s&amount=%f"
+)
```

Comment:
We don't need API requests for this exercise, The exercise is way too simple to use these for it. You can take a look at some other pull requests to see the simple implementation.

Reply:
Thanks, I was looking at the end goal of currency conversion, but this seem to be a overkill, will revert implementation.